app/controllers/subscriptions_controller.rb

```diff
@@ -10,6 +10,8 @@ def create
     else
       redirect_to root_path, error: subscription.errors.full_messages.to_sentence
     end
+  rescue ActiveRecord::RecordInvalid
+    redirect_to root_path, error: @subscription.errors.full_messages.to_sentence
   end
```

Comment:
I forgot it, sorry.
![screenshot from 2017-07-15 23-56-27](https://user-images.githubusercontent.com/17273872/28242624-44ff2332-69b9-11e7-823c-9675cf3e476a.png)